argilla/src/argilla/_models/_settings/_fields.py

```diff
@@ -31,11 +31,12 @@ class ImageFieldSettings(BaseModel):
     type: Literal["image"] = "image"
 
 
+class ChatFieldSettings(BaseModel):
+    type: Literal["chat"] = "chat"
```

Comment:
do we want to include something like `user_role`, `assitant_role`?

Reply:
I don't think that's necessary for the first iteration. But it would be nice for a later iteration where we allow the user to define the 'chat schema' based on their own data.